portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects</title>
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Work Sans' rel='stylesheet'>
+    <script src="script.js"></script>
+  </head>
+  <div class="topnav" id="topnav-projects">
+    <div class="topnav-left">
+      <a href="index.html">Sharon Dong</a>
+    </div>
+    <a href="index.html#contact">Contact</a>
+    <a href="pics.html">Fun Photos</a>
+    <a href="projects.html">Projects</a>
```

Comment:
It seems like these are repeated among each page. Is there a way to unify it?

Reply:
The specific links are the same for projects.html and about.html, but they have to be a little different in index.html. I did some quick searching and it seems that I could possibly use javascript to insert html, but I think the html code would have to be defined inside the javascript file which gets messy. Or the more popular solution seems to use php which is another language.